opensrp-app/src/main/java/org/smartregister/domain/PlanDefinition.java

```diff
@@ -133,6 +133,11 @@ public void setServerVersion(Long serverVersion) {
         this.serverVersion = serverVersion;
     }
 
+    @Override
+    public int compareTo(PlanDefinition o) {
```

Comment:
Can we leave the default comparable. Since we have implemented in database query this is not required